### PR TITLE
Adding info for Ports to be enabled on the Firewall

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**

### CODE REVIEW

File: docs/config_firewall_ports.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+title: Enable Ports on the Firewall
+sidebar_label: Enable Ports on the Firewall
+---
+
+SSR software requires that the following ports be enabled on your firewall to allow connectivity and proper operation. 
+
+### SSR to a Conductor
+
+| Service Type | Global 01 | Global 02 | Global 03 | Global 04 |
```
Comment (agrawalkaushik) [inline docs/config_firewall_ports.md:10]: The Global 01, 02 etc doesn't apply here

File: docs/config_firewall_ports.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+| --- | --- | --- | --- | --- |
+| Conductor | [Conductor IP address] | 930/TCP | 4505/TCP | 4506/TCP |
+
+
+### SSR to SSR
+
+| Service Type | Global 01 | Global 02 | Global 03 | Global 04 |
```
Comment (agrawalkaushik) [inline docs/config_firewall_ports.md:17]: same here

File: docs/config_firewall_ports.md (hunk 1 above)
Comment (MichaelBaj) [reply docs/config_firewall_ports.md:10]: Is the SSR only available in one region?

**Review by MichaelBaj — CHANGES_REQUESTED** [review]
Please update and address Kaushik's comments.

File: docs/config_firewall_ports.md (hunk 1 above)
Comment (agrawalkaushik) [reply docs/config_firewall_ports.md:10]: @MichaelBaj this one says SSR to a conductor so I did not interpret this having anything to do with MIST. Hence the comment about removing the Global01, 02 etc as that's a MIST concept.

Comment (MichaelBaj) [reply docs/config_firewall_ports.md:10]: Thanks, I must need glasses. I thought this was for the Device to Mist table. You are 100% correct. Global <x> has no meaning here.

**Review by MichaelBaj — DISMISSED** [review]

**Review by agrawalkaushik — DISMISSED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Pull request merged**